docs/tutorials/creating-a-simulation.rst

```diff
@@ -14,53 +14,36 @@ and wire them together.
 Adding Devices
 --------------
 
-In order to be included in a simulation, tickit devices must be nested within a
-`DeviceSimulation` component. At the top level, tickit simulations comprise a list of
-components - denoted in YAML by ``-``. As such we shall begin by adding an empty
-`DeviceSimulation` as the first element of our list, as:
+In order to be included in a simulation, tickit devices must have a `ComponentConfig`
+dataclass associated with them. This defines the device to be used, as well as any
+adapters to allow the device to be externally controlled. At the top level, tickit 
+simulations comprise a list of these components - denoted in YAML by ``-``.
 
-.. code-block:: yaml
-    
-    - tickit.core.components.device_simulation.DeviceSimulation:
-
-A `DeviceSimulation` wraps a device and a set of adapters, giving them a name and
-mapping the device inputs to the outputs of other devices. In this example our first
-device shall be a `RandomTrampoline` with a callback_period of :math:`1s` or
-:math:`10^9\mu\\s` named ``rand_tramp`` with no adapters - denoted in YAML by ``[]`` -
-and with no mapped inputs - denoted in YAML by ``{}``. As such we may extend our
-config, as:
+In this example our first device shall be a `RandomTrampoline` with a callback_period 
+of :math:`1s` or :math:`10^9\mu\\s` named ``rand_tramp`` with no adapters - denoted 
```

Comment:
I think you mean nanoseconds not microseconds?

```suggestion
of :math:`1s` or :math:`10^9n\s` named ``rand_tramp`` with no adapters - denoted 
```

Reply:
Yet again was in the original docs, don't blame me :eyes:

Reply:
@garryod am I just getting my prefixes wrong?

Reply:
Nah, was from back when we were in microseconds

Reply:
Forgot about that, cheers, so yes it should be microseconds @OCopping